canonical-web-and-design/maas-ui
# refactor(ui): upgrade to Vanilla 3

## Done

- Update Vanilla and react-components to Vanilla 3.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the full client with `yarn start`.
- Look around the site at different browser sizes and make sure nothing is broken (that wasn't already broken).
- Note: [this issue](https://github.com/canonical-web-and-design/react-components/issues/716) means that the mixed checkbox state is not appearing in the machine list.
